### PR TITLE
Update with mag test-dataset for centrifuge

### PR DESCRIPTION
**Pull request by sofstam**
`Samplesheet.csv` and `database.csv` were updated with mag test datasets for centrifuge

### CODE REVIEW

File: samplesheet.csv [hunk 1]
```diff
@@ -3,3 +3,4 @@ sample,run_accession,instrument_platform,fastq_1,fastq_2,fasta
 2612,ERR5766176,ILLUMINA,https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/data/fastq/ERX5474932_ERR5766176_1.fastq.gz,https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/data/fastq/ERX5474932_ERR5766176_2.fastq.gz,
 2612,ERR5766180,ILLUMINA,https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/data/fastq/ERX5474936_ERR5766180_1.fastq.gz,,
 2613,ERR5766181,ILLUMINA,https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/data/fastq/ERX5474937_ERR5766181_1.fastq.gz,https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/data/fastq/ERX5474937_ERR5766181_2.fastq.gz,
+2614,minigut,OXFORD_NANOPORE,https://github.com/nf-core/test-datasets/raw/mag/test_data/minigut_reads.fastq.gz,,
```
Comment (jfy133) [inline samplesheet.csv:6]:
```suggestion
minigut,minigut,OXFORD_NANOPORE,,,https://github.com/nf-core/test-datasets/raw/mag/test_data/minigut_reads.fastq.gz
```

Not necessary to fake the name in this instance I think :)

**Review by jfy133 — CHANGES_REQUESTED** [review]
Please also update the README to describe where the test data is from (and if how generated, if that is available on the MAG test data branch)

**Comment by sofstam** [comment]
I updated this PR.

File: README.md [hunk 2]
```diff
@@ -91,6 +92,8 @@ kraken2-build --add-to-library ../raw/NC_012920.1.fa --db taxprofiler-testdb/
 kraken2-build --build --db taxprofiler-testdb/
 kraken2-build --clean --db taxprofiler-testdb/
 ```
+#### Centrifuge
+We use the same test database for [centrifuge](https://github.com/nf-core/test-datasets/blob/mag/test_data/minigut_cf.tar.gz) as [nf-core/mag](https://github.com/nf-core/mag) is using in their test data.
 ## Support
```
Comment (jfy133) [inline README.md:97]:
```suggestion

## Support
```

File: README.md [hunk 3]
```diff
@@ -48,6 +48,7 @@ nextflow run nf-core/fetchngs --input maixner2021_acc_codes.txt --input_type sra
 ```
 FASTQ files are stored under `data/fastq/`
 
+Test data for long reads are the same as [nf-core/mag](https://github.com/nf-core/mag) is using and can be found [here](https://github.com/nf-core/test-datasets/blob/mag/test_data/minigut_reads.fastq.gz)
 ### fasta
```
Comment (jfy133) [inline README.md:52]:
```suggestion
### fasta

```

File: README.md [hunk 4]
```diff
@@ -91,6 +92,8 @@ kraken2-build --add-to-library ../raw/NC_012920.1.fa --db taxprofiler-testdb/
 kraken2-build --build --db taxprofiler-testdb/
 kraken2-build --clean --db taxprofiler-testdb/
 ```
```
Comment (jfy133) [inline README.md:94]:
```suggestion
```

```

**Review by jfy133 — APPROVED** [review]
Some markdown linting but otherwise looks good

**Comment by sofstam** [comment]
I committed your suggestions. I will merge now.

**Pull request merged**